components/post_view/post_time.jsx

```diff
@@ -43,26 +43,18 @@ export default class PostTime extends React.PureComponent {
         super(props);
 
         this.state = {
-            currentTeamDisplayName: TeamStore.getCurrent().name,
-            width: '',
-            height: ''
+            currentTeamDisplayName: TeamStore.getCurrent().name
         };
     }
 
     componentDidMount() {
         this.intervalId = setInterval(() => {
             this.forceUpdate();
         }, Constants.TIME_SINCE_UPDATE_INTERVAL);
-        window.addEventListener('resize', () => {
-            updateWindowDimensions(this);
-        });
     }
 
     componentWillUnmount() {
         clearInterval(this.intervalId);
-        window.removeEventListener('resize', () => {
-            updateWindowDimensions(this);
-        });
```

Comment:
This was added to trigger render. Going from mobile screensize to desktop resolution does not change timestamp to link.

Reply:
@hmhealey I added these listeners for resize. PR is up for review as well